packages/sync/src/utils/manageSnapshotTable.ts

```diff
@@ -12,10 +13,10 @@ const assertSessionIdIsSafe = (sessionId: string) => {
 };
 
 // includes a safety check for using in raw sql rather than via sequelize query building
-export const getSnapshotTableName = (sessionId: string) => {
+export const getSnapshotTableName = (sessionId: string, includeSchema = true) => {
```

Comment:
A bool flag in function arguments is always a potential code smell. Perhaps we should be returning `{  schema, tableName }` and letting the consumer use what it wants? Or maybe it's telling us something else? Or maybe it's telling us nothing at all, and is fine!

Reply:
It's actually not even used so I'll remove it